src/calculation/DetailedResultsBuilder.ts

```diff
@@ -304,10 +304,27 @@ export function createPatientPopulationValues(
     populationGroup.stratifier.forEach(strata => {
       if (strata.criteria?.expression) {
         const value = patientResults[strata.criteria?.expression];
+
+        // if the cqfm-appliesTo extension is present, then we want to consider the result of that
+        // population in our stratifier result
+        const appliesToExtension = strata.extension?.find(
+          e => e.url === 'http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-appliesTo'
+        );
+
+        let popValue = true;
+        if (appliesToExtension) {
+          const popCode = appliesToExtension.valueCodeableConcept?.coding?.[0].code;
+          if (popCode) {
+            popValue = patientResults[popCode];
+          }
+        }
         const result = isStatementValueTruthy(value);
+        const appliesResult = isStatementValueTruthy(value && popValue);
```

Comment:
This call to `isStatementValueTruthy` shouldn't be needed as the `&&` should be done on the `result` and the boolean result found in the `populationResults`.